ophyd/tests/test_signal.py

```diff
@@ -227,6 +236,24 @@ def test_epicssignal_readonly(cleanup, signal_test_ioc):
     time.sleep(0.2)
 
 
+def test_epicssignal_novalidation(nv_signal):
+    print('EpicsSignalNoValidation.metadata=', nv_signal.metadata)
+
+    nv_signal.put(10)
+    st = nv_signal.set(11)
+
+    time.sleep(0.05)
```

Comment:
As the waiting is happening on the object level already, it probably makes sense to remove this sleep.
```suggestion
```